app/docker/tests/testmail.sh

```diff
@@ -86,7 +91,9 @@ send() {
 		echo 'failed' > $test_results/$testname.result
 	fi
 
-	mv $test_results/mailtester $test_results/mailbox_$testname
+	#sqlite3 "$mail_db" "update mailbox set read = 1 where Metadata like '$test_str%'"
+	mail_id=$(sqlite3 "$mail_db" "select ID from mailbox where Metadata like '$test_str'")
+	curl -X PUT http://mailpit.test:8025/api/v1/messages -d "{\"IDs\":[\"$mail_id\"], \"Read\": true}"
```

Comment:
[Mailpit API](https://mailpit.axllent.org/docs/api-v1/view.html) could be used to write tests in PHP